app/server/api/user_api.py

```diff
@@ -210,10 +210,13 @@ def post(self, user_id):
 
         if reset_user_id is not None:
             user = User.query.get(reset_user_id)
-
             if user is None:
                 return make_response(jsonify({'message': 'No user found for ID: {}'.format(reset_user_id)})), 404
 
+            user.set_one_time_code(None)
+            user.password_hash = None
+            user.pin_hash = None
+
             UserUtils.admin_reset_user_pin(user)
```

Comment:
I think this new code should be in `UserUtils.admin_reset_user_pin` function

Reply:
Additionally, I think it would be nice to have the correct SMS one-time code send to the user if SMS is enabled on the instance.

Right now it sends `Dial *384*96# Safaricom or *483*46# Airtel to change your PIN`, which is valid for USSD users. Not sure how we would specify that maybe @enjeyw has some thoughts?

Reply:
Yeah agree with tristan that the lines above should just be shifted inside! TBH I think just disabling the sms is fine for now.